extension/oauth2clientextension/README.md

```diff
@@ -0,0 +1,36 @@
+# Authenticator - OIDC
```

Comment:
I believe it will be better to split the implementations into separate PRs, so it can more easily be reviewed and tracked.